app/graphql/mutations/create_user_standing.rb

```diff
@@ -0,0 +1,78 @@
+module Mutations
+  class CreateUserStanding < ApplicationQuery
+    include QueryAuthorizeSchoolAdmin
+    argument :student_id, ID, required: true
+    argument :reason, String, required: true
+    argument :standing_id, ID, required: true
+
+    description "Create a new standing log"
+
+    field :user_standing, Types::UserStandingType, null: true
+
+    def resolve(_params)
+      notify(
+        :success,
+        I18n.t("shared.notifications.done_exclamation"),
+        I18n.t("mutations.create_user_standing.success_notification")
+      )
+      { user_standing: create_user_standing }
+    end
+
+    private
+
+    def create_user_standing
+      UserStanding.transaction do
+        user_standing =
+          UserStanding.create!(
+            user_id: student.user.id,
+            reason: @params[:reason],
+            standing_id: @params[:standing_id],
+            creator: current_user
+          )
+
+        send_email
+
+        user_standing
+      end
+    end
+
+    def send_email
+      previous_standing, current_standing =
+        if user_standings.count > 1
```

Comment:
Use `exists?`

Reply:
Actually exists? Will not  work here.

Anyway, I had refactored without count in https://github.com/pupilfirst/pupilfirst/pull/1468/commits/a39d34674c2915e5bd1e0fc86c290bf070990e60